schedule/yast/textmode_installation_minimal_role/textmode_installation_minimal_role_aarch64.yaml

```diff
@@ -0,0 +1,28 @@
+---
+name:           textmode_installation_minimal_role
+description:    >
+  Full Medium installation that covers the following cases:
+    1. Installation in textmode;
+    2. "Minimal" role is selected;
+    3. Boot to command-line mode;
+    4. Installation is validated by default set of smoke tests.
+vars:
+  VIDEOMODE: text
+  SYSTEM_ROLE: minimal
+  DESKTOP: textmode
```

Comment:
we can remove these two, perhaps we need the desktop in the MR, but the role definitively not needed.